src/PHPUnit/Constraint/ResponseHasCode.php

```diff
@@ -1,13 +1,13 @@
 <?php
```

Comment:
Pokud chceme používat PHP typy, měl by tu být `declare(strict_types=1);`.

Reply:
tady typy byt musi kvuli prepsani metod z abstrakni tridy PHPUnit\Framework\Constraint\Constraint
nicmene souhlasim s povinnym nastavenim strict_types. takze bych to vsude doplnil a pridal do nastaveni cs fixeru (i u ostatnich bundlu, ktere mam ted v reseni). souhlas?

Reply:
Jj, to bych udělal, navíc to pak cs fixer může sám doplňovat pokud v rámci automatických oprav.